Qt 5.11 (error: invalid use of incomplete type 'class QButtonGroup')

Hello. Could you please fix Qt 5.11 build?

```
[   40s] + /usr/lib64/qt5/bin/qmake ImageEnlarger.pro QMAKE_STRIP= 'QMAKE_CFLAGS+=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g' 'QMAKE_CXXFLAGS+=-O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden'
[   40s] Info: creating stash file /home/abuild/rpmbuild/BUILD/smillaenlarger-0.9.0+git.2017.11.21/.qmake.stash
[   40s] + make -j3
[   40s] /usr/lib64/qt5/bin/uic src/enlargerdialog.ui -o ui_enlargerdialog.h
[   40s] /usr/lib64/qt5/bin/uic src/preferences.ui -o ui_preferences.h
[   40s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o main.o src/main.cpp
[   40s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o selectField.o src/selectField.cpp
[   40s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o previewField.o src/previewField.cpp
[   42s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o EnlargerDialog.o src/EnlargerDialog.cpp
[   43s] src/previewField.cpp: In member function 'virtual void PreviewField::paintEvent(QPaintEvent*)':
[   43s] src/previewField.cpp:175:44: warning: unused parameter 'event' [-Wunused-parameter]
[   43s]  void PreviewField::paintEvent(QPaintEvent *event) {
[   43s]                                ~~~~~~~~~~~~~^~~~~
[   44s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o Array.o src/ImageEnlargerCode/Array.cpp
[   44s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o EnlargerThread.o src/EnlargerThread.cpp
[   44s] src/ImageEnlargerCode/Array.cpp: In member function 'DirArray* MyArray::Gradient()':
[   44s] src/ImageEnlargerCode/Array.cpp:37:10: warning: variable 'p00' set but not used [-Wunused-but-set-variable]
[   44s]     float p00,p10,p20;
[   44s]           ^~~
[   44s] src/ImageEnlargerCode/Array.cpp: In member function 'void MyArray::FillWithDots()':
[   44s] src/ImageEnlargerCode/Array.cpp:79:10: warning: unused variable 'scaleF' [-Wunused-variable]
[   44s]     float scaleF = 2.0/float(sizeX);
[   44s]           ^~~~~~
[   44s] src/ImageEnlargerCode/Array.cpp: In member function 'DirArray* DirArray::Func0()':
[   44s] src/ImageEnlargerCode/Array.cpp:263:21: warning: variable 'f11' set but not used [-Wunused-but-set-variable]
[   44s]           float  f01,f11,f21;
[   44s]                      ^~~
[   45s] src/EnlargerThread.cpp: In constructor 'EnlargerThread::EnlargerThread(QObject*, int)':
[   45s] src/EnlargerThread.cpp:377:41: warning: unused parameter 'parent' [-Wunused-parameter]
[   45s]  EnlargerThread::EnlargerThread(QObject *parent, int id) {
[   45s]                                 ~~~~~~~~~^~~~~~
[   45s] In file included from src/EnlargerThread.h:34,
[   45s]                  from src/EnlargerThread.cpp:27:
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h: In instantiation of 'void BasicEnlarger<T>::WriteDstPixel(T, int, int) [with T = Point]':
[   45s] src/EnlargerThread.cpp:36:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:177:33: warning: unused parameter 'p' [-Wunused-parameter]
[   45s]     virtual void WriteDstPixel(T p, int dstCX, int dstCY)   {}
[   45s]                                ~~^
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:177:40: warning: unused parameter 'dstCX' [-Wunused-parameter]
[   45s]     virtual void WriteDstPixel(T p, int dstCX, int dstCY)   {}
[   45s]                                     ~~~~^~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:177:51: warning: unused parameter 'dstCY' [-Wunused-parameter]
[   45s]     virtual void WriteDstPixel(T p, int dstCX, int dstCY)   {}
[   45s]                                                ~~~~^~~~~
[   45s] In file included from src/EnlargerThread.cpp:32:
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h: In instantiation of 'void BasicEnlarger<T>::CalcBaseWeights() [with T = Point]':
[   45s] src/EnlargerThread.cpp:36:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:1285:25: warning: variable 'laplaceNorm' set but not used [-Wunused-but-set-variable]
[   45s]           float gradNorm,laplaceNorm;
[   45s]                          ^~~~~~~~~~~
[   45s] In file included from src/EnlargerDialog.cpp:40:
[   45s] src/CalcQueue.h: In member function 'virtual int JobListModel::rowCount(const QModelIndex&) const':
[   45s] src/CalcQueue.h:55:37: warning: unused parameter 'parent' [-Wunused-parameter]
[   45s]     int rowCount(const QModelIndex & parent = QModelIndex()) const { return jobList.size(); }
[   45s]                  ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h: In instantiation of 'void BasicEnlarger<T>::CreateKernelsFromIntegralTab(float**, int, float*, int, float, int*) [with T = Point]':
[   45s] src/EnlargerThread.cpp:36:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:971:10: warning: unused variable 'invScaleF' [-Wunused-variable]
[   45s]     float invScaleF = 1.0/scaleF;
[   45s]           ^~~~~~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:972:10: warning: unused variable 'invFineScaleF' [-Wunused-variable]
[   45s]     float invFineScaleF = (1.0/scaleF) * (1.0/float(1<<kerFineExp)) ;
[   45s]           ^~~~~~~~~~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:973:43: warning: unused variable 'k' [-Wunused-variable]
[   45s]     int smallPos,kernelStartPosFine,bigPos,k,a,tabPos;
[   45s]                                            ^
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:973:47: warning: unused variable 'tabPos' [-Wunused-variable]
[   45s]     int smallPos,kernelStartPosFine,bigPos,k,a,tabPos;
[   45s]                                                ^~~~~~
[   45s] In file included from src/EnlargerThread.h:34,
[   45s]                  from src/EnlargerThread.cpp:27:
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h: In instantiation of 'T BasicEnlarger<T>::LinModColorB(float, float, int) [with T = Point]':
[   45s] src/EnlargerThread.cpp:36:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:325:16: warning: unused variable 'faktD' [-Wunused-variable]
[   45s]     const float faktD=1.0, faktD2=0.7;  //!!! 1.0 /  0.7;
[   45s]                 ^~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h: In instantiation of 'void BasicEnlarger<T>::WriteDstPixel(T, int, int) [with T = Point4]':
[   45s] src/EnlargerThread.cpp:37:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:177:33: warning: unused parameter 'p' [-Wunused-parameter]
[   45s]     virtual void WriteDstPixel(T p, int dstCX, int dstCY)   {}
[   45s]                                ~~^
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:177:40: warning: unused parameter 'dstCX' [-Wunused-parameter]
[   45s]     virtual void WriteDstPixel(T p, int dstCX, int dstCY)   {}
[   45s]                                     ~~~~^~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:177:51: warning: unused parameter 'dstCY' [-Wunused-parameter]
[   45s]     virtual void WriteDstPixel(T p, int dstCX, int dstCY)   {}
[   45s]                                                ~~~~^~~~~
[   45s] src/EnlargerDialog.cpp: In member function 'void EnlargerDialog::slot_checkBoxChanged(int)':
[   45s] src/EnlargerDialog.cpp:390:47: warning: unused parameter 'state' [-Wunused-parameter]
[   45s]  void EnlargerDialog::slot_checkBoxChanged(int state) {
[   45s]                                            ~~~~^~~~~
[   45s] In file included from src/EnlargerThread.cpp:32:
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h: In instantiation of 'void BasicEnlarger<T>::CalcBaseWeights() [with T = Point4]':
[   45s] src/EnlargerThread.cpp:37:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:1285:25: warning: variable 'laplaceNorm' set but not used [-Wunused-but-set-variable]
[   45s]           float gradNorm,laplaceNorm;
[   45s]                          ^~~~~~~~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h: In instantiation of 'void BasicEnlarger<T>::CreateKernelsFromIntegralTab(float**, int, float*, int, float, int*) [with T = Point4]':
[   45s] src/EnlargerThread.cpp:37:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:971:10: warning: unused variable 'invScaleF' [-Wunused-variable]
[   45s]     float invScaleF = 1.0/scaleF;
[   45s]           ^~~~~~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:972:10: warning: unused variable 'invFineScaleF' [-Wunused-variable]
[   45s]     float invFineScaleF = (1.0/scaleF) * (1.0/float(1<<kerFineExp)) ;
[   45s]           ^~~~~~~~~~~~~
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:973:43: warning: unused variable 'k' [-Wunused-variable]
[   45s]     int smallPos,kernelStartPosFine,bigPos,k,a,tabPos;
[   45s]                                            ^
[   45s] src/ImageEnlargerCode/EnlargerTemplateDefs.h:973:47: warning: unused variable 'tabPos' [-Wunused-variable]
[   45s]     int smallPos,kernelStartPosFine,bigPos,k,a,tabPos;
[   45s]                                                ^~~~~~
[   45s] In file included from src/EnlargerThread.h:34,
[   45s]                  from src/EnlargerThread.cpp:27:
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h: In instantiation of 'T BasicEnlarger<T>::LinModColorB(float, float, int) [with T = Point4]':
[   45s] src/EnlargerThread.cpp:37:16:   required from here
[   45s] src/ImageEnlargerCode/EnlargerTemplate.h:325:16: warning: unused variable 'faktD' [-Wunused-variable]
[   45s]     const float faktD=1.0, faktD2=0.7;  //!!! 1.0 /  0.7;
[   45s]                 ^~~~~
[   46s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o TemplateInst.o src/TemplateInst.cpp
[   48s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o FractTab.o src/ImageEnlargerCode/FractTab.cpp
[   48s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o CalcQueue.o src/CalcQueue.cpp
[   48s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ArgumentParser.o src/ArgumentParser.cpp
[   49s] src/ArgumentParser.cpp: In member function 'virtual bool BasicOption::Parse(const QString&, const QString&, bool&)':
[   49s] src/ArgumentParser.cpp:106:65: warning: unused parameter 'nextArg' [-Wunused-parameter]
[   49s]  bool BasicOption::Parse(const QString & optArg, const QString & nextArg, bool & error) {
[   49s]                                                  ~~~~~~~~~~~~~~~~^~~~~~~
[   49s] In file included from src/CalcQueue.cpp:34:
[   49s] src/CalcQueue.h: In member function 'virtual int JobListModel::rowCount(const QModelIndex&) const':
[   49s] src/CalcQueue.h:55:37: warning: unused parameter 'parent' [-Wunused-parameter]
[   49s]     int rowCount(const QModelIndex & parent = QModelIndex()) const { return jobList.size(); }
[   49s]                  ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
[   49s] src/CalcQueue.cpp: In member function 'virtual bool JobListModel::insertRows(int, int, const QModelIndex&)':
[   49s] src/CalcQueue.cpp:77:75: warning: unused parameter 'parent' [-Wunused-parameter]
[   49s]  bool JobListModel::insertRows(int position, int rows, const QModelIndex & parent) {
[   49s]                                                        ~~~~~~~~~~~~~~~~~~~~^~~~~~
[   49s] src/CalcQueue.cpp: In member function 'virtual bool JobListModel::removeRows(int, int, const QModelIndex&)':
[   49s] src/CalcQueue.cpp:86:75: warning: unused parameter 'parent' [-Wunused-parameter]
[   49s]  bool JobListModel::removeRows(int position, int rows, const QModelIndex & parent) {
[   49s]                                                        ~~~~~~~~~~~~~~~~~~~~^~~~~~
[   49s] src/CalcQueue.cpp: In member function 'void CalcQueue::SearchRemoveAtEnd()':
[   49s] src/CalcQueue.cpp:678:9: warning: variable 'found' set but not used [-Wunused-but-set-variable]
[   49s]     bool found = false;
[   49s]          ^~~~~
[   49s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ConsoleManager.o src/ConsoleManager.cpp
[   49s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ED_LoadSave.o src/ED_LoadSave.cpp
[   50s] src/ConsoleManager.cpp: In member function 'bool ConsoleManager::TryOpenSource(QString, QImage&)':
[   50s] src/ConsoleManager.cpp:204:9: warning: variable 'isSymLink' set but not used [-Wunused-but-set-variable]
[   50s]     bool isSymLink = false;
[   50s]          ^~~~~~~~~
[   51s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ED_Parameter.o src/ED_Parameter.cpp
[   52s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o Preferences.o src/Preferences.cpp
[   52s] In file included from src/ED_LoadSave.cpp:39:
[   52s] src/CalcQueue.h: In member function 'virtual int JobListModel::rowCount(const QModelIndex&) const':
[   52s] src/CalcQueue.h:55:37: warning: unused parameter 'parent' [-Wunused-parameter]
[   52s]     int rowCount(const QModelIndex & parent = QModelIndex()) const { return jobList.size(); }
[   52s]                  ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
[   53s] g++ -c -pipe -O2 -Wall -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -g -fvisibility=hidden -fvisibility-inlines-hidden -O2 -Wall -W -D_REENTRANT -fPIC -DAPP_VERSION=\"0.9.0+git.2017.11.21\" -DQT_DEPRECATED_WARNINGS -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt5 -isystem /usr/include/qt5/QtWidgets -isystem /usr/include/qt5/QtGui -isystem /usr/include/qt5/QtCore -I. -isystem /usr/include/libdrm -I. -I/usr/lib64/qt5/mkspecs/linux-g++ -o ClipRect.o src/ClipRect.cpp
[   53s] In file included from src/ED_Parameter.cpp:37:
[   53s] src/CalcQueue.h: In member function 'virtual int JobListModel::rowCount(const QModelIndex&) const':
[   53s] src/CalcQueue.h:55:37: warning: unused parameter 'parent' [-Wunused-parameter]
[   53s]     int rowCount(const QModelIndex & parent = QModelIndex()) const { return jobList.size(); }
[   53s]                  ~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
[   53s] src/ED_Parameter.cpp: In member function 'void EnlargerDialog::slot_ParamCheckBoxChanged(int)':
[   53s] src/ED_Parameter.cpp:349:52: warning: unused parameter 'state' [-Wunused-parameter]
[   53s]  void EnlargerDialog::slot_ParamCheckBoxChanged(int state) {
[   53s]                                                 ~~~~^~~~~
[   54s] src/Preferences.cpp: In constructor 'PreferencesDialog::PreferencesDialog(QWidget*)':
[   54s] src/Preferences.cpp:40:33: error: invalid use of incomplete type 'class QButtonGroup'
[   54s]     typeGroup = new QButtonGroup();
[   54s]                                  ^
[   54s] In file included from /usr/include/qt5/QtWidgets/qradiobutton.h:44,
[   54s]                  from /usr/include/qt5/QtWidgets/QRadioButton:1,
[   54s]                  from ./ui_preferences.h:20,
[   54s]                  from src/Preferences.cpp:31:
[   54s] /usr/include/qt5/QtWidgets/qabstractbutton.h:53:7: note: forward declaration of 'class QButtonGroup'
[   54s]  class QButtonGroup;
[   54s]        ^~~~~~~~~~~~
[   54s] src/Preferences.cpp:41:13: error: invalid use of incomplete type 'class QButtonGroup'
[   54s]     typeGroup->addButton(uiPref->srcTypeRadioButton);
[   54s]              ^~
[   54s] In file included from /usr/include/qt5/QtWidgets/qradiobutton.h:44,
[   54s]                  from /usr/include/qt5/QtWidgets/QRadioButton:1,
[   54s]                  from ./ui_preferences.h:20,
[   54s]                  from src/Preferences.cpp:31:
[   54s] /usr/include/qt5/QtWidgets/qabstractbutton.h:53:7: note: forward declaration of 'class QButtonGroup'
[   54s]  class QButtonGroup;
[   54s]        ^~~~~~~~~~~~
[   54s] src/Preferences.cpp:42:13: error: invalid use of incomplete type 'class QButtonGroup'
[   54s]     typeGroup->addButton(uiPref->otherTypeRadioButton);
[   54s]              ^~
[   54s] In file included from /usr/include/qt5/QtWidgets/qradiobutton.h:44,
[   54s]                  from /usr/include/qt5/QtWidgets/QRadioButton:1,
[   54s]                  from ./ui_preferences.h:20,
[   54s]                  from src/Preferences.cpp:31:
[   54s] /usr/include/qt5/QtWidgets/qabstractbutton.h:53:7: note: forward declaration of 'class QButtonGroup'
[   54s]  class QButtonGroup;
[   54s]        ^~~~~~~~~~~~
[   54s] src/Preferences.cpp: In destructor 'virtual PreferencesDialog::~PreferencesDialog()':
[   54s] src/Preferences.cpp:53:11: warning: possible problem detected in invocation of delete operator: [-Wdelete-incomplete]
[   54s]     delete typeGroup;
[   54s]            ^~~~~~~~~
[   54s] src/Preferences.cpp:53:11: warning: invalid use of incomplete type 'class QButtonGroup'
[   54s] In file included from /usr/include/qt5/QtWidgets/qradiobutton.h:44,
[   54s]                  from /usr/include/qt5/QtWidgets/QRadioButton:1,
[   54s]                  from ./ui_preferences.h:20,
[   54s]                  from src/Preferences.cpp:31:
[   54s] /usr/include/qt5/QtWidgets/qabstractbutton.h:53:7: note: forward declaration of 'class QButtonGroup'
[   54s]  class QButtonGroup;
[   54s]        ^~~~~~~~~~~~
[   54s] src/Preferences.cpp:53:11: note: neither the destructor nor the class-specific operator delete will be called, even if they are declared when the class is defined
[   54s]     delete typeGroup;
[   54s]            ^~~~~~~~~
[   54s] make: *** [Makefile:693: Preferences.o] Error 1
[   54s] make: *** Waiting for unfinished jobs....
```

--- a/src/Preferences.cpp
+++ b/src/Preferences.cpp
@@ -21,6 +21,7 @@ You should have received a copy of the GNU General Public License
 along with this program; if not, see <http://www.gnu.org/licenses/>.
 
 ---------------------------------------------------------------------- */
+#include <QButtonGroup>
 #include <QFileDialog>
 #include <QDir>
 #include <QSettings>

